lib/galaxy/tools/deps/resolvers/conda.py

```diff
@@ -31,7 +31,7 @@
 
 DEFAULT_BASE_PATH_DIRECTORY = "_conda"
 DEFAULT_CONDARC_OVERRIDE = "_condarc"
-DEFAULT_ENSURE_CHANNELS = "conda-forge,r,bioconda,iuc"
+DEFAULT_ENSURE_CHANNELS = "iuc,bioconda,r,defaults,conda-forge"
```

Comment:
I'm running on Linux and using .condarc.  I'm still running miniconda2, version 4.2.13.  I have not tried the ```conda config --add channels``` approach, so I'll try it and see if it makes a difference.  Thanks!